usr/lib/api/api_interface.c

```diff
@@ -286,7 +286,31 @@ void child_fork_initializer()
     if (Anchor != NULL)
         C_Finalize(NULL);
     in_child_fork_initializer = FALSE;
- }
+}
+
+void parent_fork_prepare()
+{
+    if (Anchor == NULL)
+        return;
+
+    /*
+     * Stop the event thread in the fork parent, since having the event thread
+     * active when a fork is performed causes various problems (e.g. deadlocks
+     * in glibc).
+     */
+    if (Anchor->event_thread > 0)
```

Comment:
Is it save to assume that it will be the same on all Linuxes? I don't think OCK will compile on non-Linux platforms anyway. So I would not spend much effort to make it portable to non-Linux platforms as of now.

Reply:
Good question.  I do not know the answer.  But I would assume it is the same on all Linuxes.  Just my assumption though.  Standard does not distinguish between Linux and other POSIX OSes.

Reply:
At least in travis it builds on s390, x86, arm, and power.